acceptance/openstack/compute/v2/bootfromvolume_test.go

```diff
@@ -303,3 +303,46 @@ func TestBootFromNewCustomizedVolume(t *testing.T) {
 
 	tools.PrintResource(t, server)
 }
+
+func TestBootFromNewVolumeWithTag(t *testing.T) {
```

Comment:
Have you considered adding a tag to the volume create request of `TestBootFromNewVolume()` rather than creating a new test? I'm asking, because the drawback is a new test to maintain, that has a lot of duplication with a existing test.

Reply:
I just thought adding a new test for new functionality was best. I'm happy to modify the existing test if you prefer.

Reply:
I'd prefer we reuse the existing test yeah, if you don't mind changing.

Reply:
The reason I added a new test is because we're forcing a microversion so we can get tags from volume attachments.
I'm just afraid that we might miss something with more recent microversions if we don't isolate that test.

Reply:
I'll mutualise so we can move forward.